src/components/button/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link'
+import validator from 'validator'
+import { isEmpty, isNull } from 'lodash'
+
+export type ButtonProps = {
+  id: string
+  label: string
+  url: string
+}
+
+function isRequestValid(button) {
```

Comment:
What about naming this `isRequestInvalid`, since we are returning out of the component if the function returns `true`? 

Or, keep the name and invert some of the logic:
```
!isEmpty(button.label) && !isEmpty(button.url) && validator.isURL(button.url)
```
maybe?